app/views/activities/index.html.erb

```diff
@@ -45,7 +45,7 @@
       </tbody>
     </table>
 
-  <%= link_to t('activities.choose_next_activity'), activity_categories_path, class: 'btn btn-success' if can? :suggest_activity, @school %>
```

Comment:
This is an unrelated change, but this allowed me to remove an otherwise unused ability.